initial Eltex-mes IPv6 address discovery

Npeca75 opened the pull request:
Eltex-mes OS using Radlan MIBs
so half of information is in IP-MIB, other half in RADLAN-IPv6

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)


#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

Npeca75 commented:
![lnms1](https://user-images.githubusercontent.com/59750439/144546092-25bf46a6-af50-406e-acbb-891c883f933f.png)
![lnms2](https://user-images.githubusercontent.com/59750439/144546093-371a635c-8b97-404f-92ad-3868d96b2e9e.png)

Jellyfrog commented:
mibs should be splitted by vendor instead of os/device

Npeca75 commented:
> mibs should be splitted by vendor instead of os/device

Hi @Jellyfrog 
last time? few month ago? if i remember correctly, PipoCanaja stated that YAML could handle only one "mib dir"

    SNMP['/usr/bin/snmpwalk' '-M' '/opt/librenms/mibs:/opt/librenms/mibs/eltexmes23xx' '-v2c' '-c' 'COMMUNITY' '-OQXUte' 'udp:HOSTNAME:161' 'RADLAN-IPv6::rlIpAddressTable']  

as you see, MIBs are taken from mibs/xxxxx dir

if you have other suggestion could you please post an example ?

murrant commented on includes/discovery/ipv6-addresses/eltex-mes21xx.inc.php:
```diff
@@ -0,0 +1,46 @@
+//standard MIB
+$oids = SnmpQuery::walk('IP-MIB::ipAddressIfIndex.ipv6')->table(2);
+//Radlan MIB
+$oidr = SnmpQuery::walk('RADLAN-IPv6::rlIpAddressTable')->table(2);
```
You can merge the two arrays by adding the $oids variable as the second argument of ->table().  Will make things easier below.

Npeca75 replied:
done

Npeca75 commented:
> mibs should be splitted by vendor instead of os/device

ok, moved IP and IPv6 MIB in /mibs/radlan and added group: radlan in two Eltex OS
other MIBs could not be simply merged, because they are sooo different, furthermore, mes21 fail discovery with mes23 MIBs and vice versa, so i left them separate. will see if checks fails

murrant commented on includes/discovery/ipv6-addresses/eltex-mes21xx.inc.php:
```diff
@@ -0,0 +1,46 @@
+
+        $index = $iparray['IP-MIB::ipAddressIfIndex'];
+        $prefixlen = $oidr['ipv6'][$ip]['RADLAN-IPv6::rlIpAddressPrefixLength'];
+        $split = str_split(str_replace(':', '', strtolower($ip)), 4); //convert colon delimited hex IPv6 address to array, every forth char
```
We have an IPv6 class to help with ip validation/manipulation.

Npeca75 replied:
ah, indeed ... :slightly_smiling_face: 
done

murrant reviewed (approved):
Immaculate :)
